gtsam/linear/NoiseModel.h

```diff
@@ -207,12 +207,25 @@ namespace gtsam {
       virtual Vector unwhiten(const Vector& v) const;
 
       /**
-       * Mahalanobis distance v'*R'*R*v = <R*v,R*v>
+       * Squared Mahalanobis distance v'*R'*R*v = <R*v,R*v>
        */
-      virtual double Mahalanobis(const Vector& v) const;
+      virtual double SquaredMahalanobisDistance(const Vector& v) const;
```

Comment:
Actually, since this is not a static function, should start with lowercase ie “squaredMahalanobisDistance”

Reply:
Done